hello-world.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+echo "Hello World";
```

Comment:
According to the style guide there should be an EOF comment and a new line at the end of the file

Reply:
Thanks!